ANGULAR.md

```diff
@@ -0,0 +1,198 @@
+# Angular DX Check
+
+## Onboarding
+
+I did `npm install -g @angular/cli`. I made a new app and a simple component - works great. I included SCSS and Routing.
+
+Onboarding worked as expected - all the files were generated in the correct places.
+
+## First Test
+
+I wrote a simple test for my component `<Greeter />`:
+
+```ts
+import { Component, OnInit } from '@angular/core';
+
+@Component({
+  selector: 'greeter',
+  templateUrl: './greeter.component.html',
+  styleUrls: ['./greeter.component.scss']
+})
+export class GreeterComponent implements OnInit {
+  count = 5
+
+  constructor() { }
+
+  ngOnInit(): void {
+  }
+
+  increment () {
+    this.count++
+  }
+}
+```
+
+And added a test:
+
+```ts
+import { GreeterComponent } from './greeter.component'
+
+describe('greeter.component.cy.ts', () => {
+  it('playground', () => {
+    cy.mount(GreeterComponent)
+  })
+})
+```
+
+Also works great. `cy.mount` has no squigglies - the types were found correctly (surprisingly; this is often not the case). The correct types for `describe` were also found (no conflict with Jest, etc - since Angular does not use Jest).
+
+
+# Errors
+
+## User App Error
+
+I added some invalid code:
+
+```ts
+import { Component, OnInit } from '@angular/core';
+
+  let foo bar '' 

```

Comment:
I think this is classified as a `typing error` since you can still make source code changes and have those changes apply even with this error e.g. webpack is still successfully emitting assets